mdbook-tera-backend/src/main.rs

```diff
@@ -0,0 +1,29 @@
+mod tera_renderer;
+
+use mdbook::renderer::RenderContext;
+use std::io;
+
+use crate::tera_renderer::*;
```

Comment:
Please avoid glob imports — they make it hard to review the code in places like GitHub.